govc/vm/change.go

```diff
@@ -53,10 +54,38 @@ func init() {
 	cli.Register("vm.change", &change{})
 }
 
+// setAllocation sets *info=nil if none of the fields have been set.
+// We need non-nil fields for use with flag.FlagSet, but we want the
+// VirtualMachineConfigSpec fields to be nil if none of the related flags were given.
+func setAllocation(info **types.ResourceAllocationInfo) {
+	r := *info
+
+	if r.Shares.Level == "" {
+		r.Shares = nil
+	} else {
+		return
+	}
+
+	if r.Limit != nil {
+		return
+	}
+
+	if r.Reservation != nil {
+		return
+	}
+
```

Comment:
do we care `ExpandableReservation` here?

Reply:
Good catch.  It seems `ExpandableReservation` cannot be set to `true` for the VM fields.  I'll see about just excluding the `{cpu,mem}.expandable` flags from the vm.change command.

Reply:
Yes please exclude, bc it´s a resource pool flag.

Reply:
ok done.